src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working");
```

Comment:
The message in `System.out.println` should indicate the type of machine. Consider changing the message to something more informative like 'Bulldozer is doing its work'.